### PR TITLE
Scatter plot for vector outputs

### PR DESCRIPTION
**Pull request by pzhanggit**

### CODE REVIEW

File: hydragnn/postprocess/visualizer.py [hunk 1]
```diff
@@ -636,12 +687,13 @@ def create_scatter_plots(
     ):
         """Creates scatter plots for all head predictions."""
         for ihead in range(self.num_heads):
-            if self.head_dims[ihead] == 3:
+            if self.head_dims[ihead] > 1:
                 # vector output
-                self.create_scatter_plot_nodes_vec(
```
Comment (streeve) [inline hydragnn/postprocess/visualizer.py:641]: What happens for vector node properties?

Comment (pzhanggit) [reply hydragnn/postprocess/visualizer.py:641]: It is still there. We do not use it for now. It was designed specifically for node feature vector with three components like magnetic moment in alloy datasets.

Comment (streeve) [reply hydragnn/postprocess/visualizer.py:641]: I misunderstood this last week. Planning to push naming changes and then I think this is ready to merge

**Review by streeve — APPROVED** [review]
Ready to merge if you are happy with the new function names

**Pull request merged**